store/tikv/mock-tikv/cluster.go

```diff
@@ -34,15 +37,15 @@ type Cluster struct {
 	sync.RWMutex
 	id      uint64
 	stores  map[uint64]*Store
-	regions map[uint64]*Region
+	Regions map[uint64]*Region
```

Comment:
Exporting it will make `Cluster` not thread safe. How about add a new method `dumpRegions() []*Region`?